product_docs/docs/pem/9/installing_pem_agent/index.mdx

```diff
@@ -1,35 +1,49 @@
 ---
-navTitle: "Installing the PEM agent"
-title: "Installing Postgres Enterprise Manager agent"
-# This is new created file
-# Line 15 onwards need review/edits
+navTitle: Installing
```

Comment:
For the navTitle, we should differentiate between installing the agent and installing the server. You can see how the old version was "Installing the PEM agent". Right now, for PEM, the nav tree just shows two "Installing" entries.

Reply:
Added a block for these - also overriding the defaults in cases where we include Windows or MacOS instructions